requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy
 scipy
 pandas>=0.19
-dill
\ No newline at end of file
+dill==0.2.7.1
```

Comment:
No, I had two failing tests caused by the version of `dill` I had installed. I upgraded to the newest version and it was fixed, so I figured I'd pin it to known working version. I'll unpin. :) In general, I'm curious why you prefer unpinned?